src/NuGet.Core/NuGet.PackageManagement/Resolution/ResolverGather.cs

```diff
@@ -172,6 +178,7 @@ public int MaxDegreeOfParallelism
                 // We are done when the queue is empty, and the number of finished requests matches the total request count
                 if (_gatherRequests.Count < 1 && _workerTasks.Count < 1)
                 {
+                    _context.Log.LogDebug(string.Format("Total number of results gathered : {0}", _results.Count));
```

Comment:
Discussed with Yishai, ndebug messages should be in english only so no need to localize them. But i'll localize anything which is Info or minimal.